### DISCUSSION TITLE
JSON Format Error while compiling the evaluator in the container

### DISCUSSION THREAD

**Discussion by ali-ihsan-gullu** [christianbirchler-org/sdc-testing-competition#10] [General]
The JSON file in the "Evaluator" folder is provided in the "Git - Large File System" format. The following command must be executed to run the evaluator:

`$ sudo git lfs pull `

If git lfs is not installed, it can be installed using the following command:
```
$ curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
$ curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.python.sh | bash
$ git lfs --version
```

**Comment by ChristianBirchler** [comment]
Thanks @vatozZ for this suggestion. I will mention this in the docs.

**Comment by ChristianBirchler** [comment]
Addressed in #12.
